__tests__/app.test.js

```diff
@@ -86,4 +87,70 @@ describe('testing GET /api/articles/:article_id', () => {
             expect(data._body.msg).toBe("Invalid input: 400 Bad Request");
         })
     });
-});
\ No newline at end of file
+});
+
+describe('testing GET /api/articles', () => {
+    test('should respond with status 200 and an articles array of all article objects each with the required properties', () => {
+
+        // make sure body property is not included in final list of objects
+
+        return request(app).get("/api/articles")
+        .expect(200)
+        .then(response => {
+            expect(response._body.articles.length > 0).toEqual(true); // check for empty arr
+            response._body.articles.forEach((article => {
+                expect(typeof article.author).toBe("string")
+                expect(typeof article.title).toBe("string")
+                expect(typeof article.article_id).toBe("number")
+                expect(typeof article.topic).toBe("string")
+                expect(typeof article.created_at).toBe("string")
+                expect(typeof article.votes).toBe("number")
+                expect(typeof article.article_img_url).toBe("string")
+                // will test for comment_count in next test
+            }))
+        })
+    });
```

Comment:
Good to see you're testing the shape of the data here. I'd be tempted just check the `comment_count` here also rather than a seperate test